profiles/mappings/default.cfg

```diff
@@ -185,24 +203,28 @@ OWASP_V3: OWASP-SM-005_____Testing for CSRF
 OWASP_V4: OTG-SESS-005_____Testing for Cross Site Request Forgery
 NIST: SC-23_____Session Authenticity - CSRF
 OWASP_TOP_10: A8_____Cross-Site Request Forgery
+CWE: CWE-352_____Cross-Site Request Forgery
 
 [OWTF-AZ-001]
 OWASP_V3: OWASP-AZ-001_____Testing Directory traversal/file include
 OWASP_V4: OTG-AUTHZ-002_____Testing Directory traversal/file include
 NIST: AC-6_____Least Privilege - Directory traversal
-OWASP_TOP_10: A5_____Security Misconfiguration - Directory traversal
+OWASP_TOP_10: A5_____Insecure Direct Object References
+CWE: CWE-22_____Path Traversal
 
```

Comment:
@DarKnight-- , 22 is child of OWASP Top Ten 2013 Category A4 - Insecure Direct Object References. [1]

[1] http://cwe.mitre.org/data/definitions/932.html